quickstart.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
```

Comment:
In general might help to run this through `shellcheck` (same goes for any other `*.sh` script)

Reply:
Good suggestions, thanks. No major issues were there, improved a few lines according to the findings. Will run that once we have all components in place.